cxplat/src/cxplat_winuser/cxplat_winuser.cpp

```diff
@@ -152,6 +158,7 @@ cxplat_initialize()
         if (leak_detector) {
             _cxplat_leak_detector_ptr = std::make_unique<cxplat_leak_detector_t>();
         }
+#endif
 
         cxplat_status_t status = cxplat_winuser_initialize_thread_pool();
```

Comment:
Does this `throw`? If not, is there an overhead of surrounding it with a `try`/`catch` when it can't throw?